build.sbt

```diff
@@ -1,15 +1,9 @@
 val catsVersion = "2.7.0"
-val finagleVersion = "21.8.0"
+val finagleVersion = "22.4.0"
```

Comment:
I merged in https://github.com/typelevel/catbird/pull/454 because MiMa needed a version bump anyway.